### ISSUE TITLE
+ / - buttons to open and collapse regions does not work with NVDA screen reader

### ISSUE THREAD

**Issue by HerinHentry** [w3c/wai-website#32]
Hi,

Looking into the page https://www.w3.org/WAI/beta/planning-and-managing/initiate/.
The + and - works fine with a mouse, keyboard. But, when NVDA is on it does not work. Checked in Chrome.

Can someone please try to reproduce the same in JAWS?

Thanks

**Comment by yatil** [comment]
I was able to open the section using NVDA in Chrome (Windows 8.1), but not to close it. Need to closer investigate.

**Comment by charadani** [comment]
In my testings, the only way in which this components work reliably while NVDA is in use is by pressing the NVDA+F2 combination, which passes keyboard control back to the browser. Whenever this keystroke is pressed, or the screen reader is in focus mode, the component works reliably: it expands or collapse when The Enter or the Space key is pressed.

I tested it with NVDA 2018.1.1 and Chrome  and Google Chrome Versión 66.0.3359.139

**Comment by yatil** [comment]
@charadani Thanks for the feedback – Do you think that’s known to NVDA users? I have the feeling that websites can’t do much if NVDA doesn’t pipe the keystroke through to the browser.

**Comment by yatil** [comment]
We decided to push this back to tackle once we have a better understanding of why NVDA is doing what it's doing. Currently, we are not getting any keyboard events when not in focus mode, so there is nothing that we can do right now.

**Comment by yatil** [comment]
I asked a screen reader user to give feedback on this.

**Comment by yatil** [comment]
We did not get any more feedback about this, so it seems to be usable. I’ll keep an eye on user feedback and see if it turns out to be a problem.
